Studio-Lovelies/GG-JointJustice-Unity
# Remove checkmarks from examined `Move` choices

## Summary
`Move` choices currently receive checkmarks after being selected, just like `Talk` choices.
To mimic the behavior of the original games, this PR fully removes checkmarks from `Move` choices.

## Before/after screenshots and/or animated gif

### Before

https://github.com/user-attachments/assets/5a614e13-11a7-4dc4-bd9c-fb27560383c2

### After

> [!NOTE]  
> As I select `Move` for a second time, you will see **one** frame where Unity shows the "unknown location" preview image.  
> This is a known issue and I will attempt to resolve it as part of `extend the ActionDecoder to pre-load Move scenes, rather than loading them on demand.` in #456.

https://github.com/user-attachments/assets/103fa5d2-69d1-4614-9e5a-f741a5cea83c


## Testing instructions
1. Open the `Game` scene
2. Select `Move`
3. Select `Kitchen`
4. Play through the sub-story
5. Select `Move` again
6. Notice the preview image changing, but the label not receiving a checkmark (like `Talk` choices would)

Following these steps on `develop` should still show a checkmark.

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - #456
